update.sh

```diff
@@ -18,100 +18,98 @@ arch=$(get_arch)
 
 yarnVersion="$(curl -sSL --compressed https://yarnpkg.com/latest-version)"
 
-function update_node_version {
-
-	local baseuri=$1
-	shift
-	local version=$1
-	shift
-	local template=$1
-	shift
-	local dockerfile=$1
-	shift
-	local variant=
```

Comment:
I had issues with this not having the equal in another PR, did it trigger an error in shfmt?

Reply:
I was going to comment that `local variant=` and `local variant` have exactly the same behavior, but it just dawned on me that they don't. Will fix :)

Reply:
https://github.com/mvdan/sh/issues/239

Reply:
Nice!! It's not just me then! Thanks!

Stellar support btw!